src/js/hca/config/form.js

```diff
@@ -25,6 +25,7 @@ import {
 } from '../helpers';
 
 import IntroductionPage from '../components/IntroductionPage';
+// import IntroductionPage from '../components/SIPIntroductionPage';
```

Comment:
Yeah, when we get all these PRs merged into master, I'll put it behind a feature flag so we can test it in staging without putting it out on production.